reference/platform-app-yaml.md

```diff
@@ -390,8 +398,10 @@ relationships:
     solr: "solr:solr"
     redis: "redis:redis"
 web:
-    document_root: "/"
-    passthru: "/index.php"
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
```

Comment:
One of them is a URL, these start with `/`. The other is a path on disk relative to the application root, these do not.